package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "lab-03-fs-async",
+  "version": "1.0.0",
+  "description": "lab-03",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha, chai"
```

Comment:
This should just be "mocha" not "mocha, chai" it literally runs the words you specify here as a command when you run 'npm test' in your lab-root directory.  Running 'mocha, chai' in your terminal obviously doesn't provide the expected result.